docs/docs/multi-dc.md

```diff
@@ -0,0 +1,176 @@
+```
+
+1. Ensure there are TCP routes allowing for traffic flow between each of the Mesos clusters.
+2. Start each cluster with the necessary configuration.
+3. Wait for each cluster to allocate it's seed nodes.
+4. After the seeds have been allocated a restart needs to be issued to all of the nodes that have already been started. Send a `POST` to the http API at `/cluster/rolling-restart/start` and the framework will restart each of the nodes so that new new seed set will take effect.
+5. Your clusters should now be able to replicate data between each other.
+
+## Example on DCOS
+
+Imagine three Data Centers dcos0, dcos1, dcos2 running three different DCOS Clusters.
+
+The following options files would be used for the install for each instance of Cassandra on DCOS
+
+###### dcos0.json
+```
+{
+  "cassandra": {
+    "cluster-name": "multi-dc",
+    "resources": {
+      "cpus": 1.0,
+      "mem": 2048,
+      "disk": 6144
+    },
+    "dc": {
+      "default-dc": "dcos0",
+      "default-rack": "rack0",
+      "external-dcs": [
+        {
+          "name": "dcos1",
+          "url": "http://dcos1/service/cassandra.multi-dc"
+        },
+        {
+          "name": "dcos2",
+          "url": "http://dcos2/service/cassandra.multi-dc"
+        }
+      ]
+    }
+  }
+}
+```
+
+###### dcos1.json
+```
+{
+  "cassandra": {
+    "cluster-name": "multi-dc",
+    "resources": {
+      "cpus": 1.0,
+      "mem": 2048,
+      "disk": 6144
+    },
+    "dc": {
+      "default-dc": "dcos1",
+      "default-rack": "rack1",
+      "external-dcs": [
+        {
+          "name": "dcos0",
+          "url": "http://dcos0/service/cassandra.multi-dc"
+        },
+        {
+          "name": "dcos2",
+          "url": "http://dcos2/service/cassandra.multi-dc"
+        }
+      ]
+    }
+  }
+}
+```
+
+###### dcos2.json
+```
+{
+  "cassandra": {
+    "cluster-name": "multi-dc",
+    "resources": {
+      "cpus": 1.0,
+      "mem": 2048,
+      "disk": 6144
+    },
+    "dc": {
+      "default-dc": "dcos2",
+      "default-rack": "rack2",
+      "external-dcs": [
+        {
+          "name": "dcos0",
+          "url": "http://dcos0/service/cassandra.multi-dc"
+        },
+        {
+          "name": "dcos1",
+          "url": "http://dcos1/service/cassandra.multi-dc"
+        }
+      ]
+    }
+  }
+}
+```
+
+1. On DCOS Cluster 0 install with `dcos package install cassandra --options=dcos0.json`
+2. On DCOS Cluster 1 install with `dcos package install cassandra --options=dcos1.json`
+3. On DCOS Cluster 2 install with `dcos package install cassandra --options=dcos2.json`
+4. Wait for each cluster to allocate it's seed nodes.
+5. After the seeds have been allocated a restart needs to be issued to all of the nodes that have already been started. Send a `POST` to the http API at `/cluster/rolling-restart/start` and the framework will restart each of the nodes so that new new seed set will take effect.
```

Comment:
Change to: "After the seeds have been allocated, you must restart all of the nodes that have already been started. "